### DISCUSSION TITLE
Solved error: PGPT_PROFILES=local make run

### DISCUSSION THREAD

**Discussion by Zirgite** [imartinez/privateGPT#1141] [Show and tell]
I had the following error on windows 10:
PGPT_PROFILES=local make run
PGPT_PROFILES=local : The term 'PGPT_PROFILES=local' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is 
correct and try again.
At line:1 char:1      
+ PGPT_PROFILES=local make run
+ ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PGPT_PROFILES=local:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
    
    it was solved by:
    $env:PGPT_PROFILES = "local"
     make run

COMMENT:  I was trying to run the command PGPT_PROFILES=local make run on a Windows platform using PowerShell. The syntax VAR=value command is typical for Unix-like systems (e.g., Linux, macOS) and won't work directly in Windows PowerShell.

To resolve this issue, I needed to set the environment variable differently in PowerShell and then run the command.
SOLUTION:
    $env:PGPT_PROFILES = "local"
     make run

**Comment by ajatau** [comment]
I faced the same problem but now i face the same issue while trying to run 

`make run`

**Reply by Zirgite** [reply]
So was it solved or not?

**Comment by ajatau** [comment]
ResolvedAbba BardeOn 1 Nov 2023, at 12:38, TheodorC ***@***.***> wrote:﻿
So was it solved or not?

**Comment by SamNeblett** [comment]
I'm still running into this error. Setting the environment variable did not help.

**Reply by champ007644** [reply]
did you solve this? same problem

**Comment by ajatau** [comment]
@SamNeblett set the environment variable for PGPT_PROFILES as local

`$env:PGPT_PROFILES="local"`
Then 
run `poetry run python -m private_gpt`

This would work

**Reply by SamNeblett** [reply]
This is still not working for me... Even after manually setting the environment variable.

**Reply by Seeders** [reply]
Yep, that worked for me.

`$env:PGPT_PROFILES="local"`

Then

`poetry run python -m private_gpt`

Then just visit

http://localhost:8001/

**Reply by Seeders** [reply]
Alternatively you can use

`choco install make`

then 

`make run`

**Comment by nic0711** [comment]
I found a workaround in deactivation the Metal GPU "CMAKE_ARGS="-DLLAMA_METAL=**off**""
It runs, but slow.

Any solution for Apple M2Max?
